src/index.html

```diff
@@ -5,8 +5,379 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Estonia&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+              href="#"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-block body__main-block">
+      <div class="assortment-of-devices main-block__assortment-of-devices">
+
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="Product image"
+          >
+
+          <div class="card__product-data">
+            <p class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+            <p class="card__product-code">Product code: 195434</p>
+          </div>
+
+          <div class="card__stars-and-reviews card__stars-and-reviews--top-gap">
+            <div class="stars-container">
+              <div class="stars-container__stars stars-container__stars--4">
+                <div class="stars-container__star"></div>
+                <div class="stars-container__star"></div>
+                <div class="stars-container__star"></div>
+                <div class="stars-container__star"></div>
+                <div class="stars-container__star"></div>
+              </div>
+            </div>
```

Comment:
`stars-container` is redudant wrapper. Fix all cards
```suggestion
              <div class="stars stars--4">
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
              </div>
```